src/util/tracy_wrapper.h

```diff
@@ -0,0 +1,200 @@
+/*
+Minetest
+Copyright (C) 2024 DS
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
```

Comment:
#14439  SPDX headers could be used here to cut the bloat.

Reply:
I'd prefer to have all files changed at once, because consistency is beautiful.

Reply:
I thought we could do it the same way as code style guidelines are enforced - by correcting issues when they're modified by a diff.
Not that it matters much; I'll leave the decision up to you.